sites/geohub/src/routes/(app)/data/supported-formats/+page.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+	import { AccepedExtensions } from '$lib/config/AppConfig';
+</script>
+
+<p class="title is-4 has-text-centered mt-5">Supported Formats</p>
+<div class="table-container mt-5">
+	<table class="table fullwidth-table ml-auto mr-auto small striped">
+		<thead class="table-header">
+			<tr>
+				<th>Name</th>
+				<th>extensions</th>
+			</tr>
+		</thead>
+		<tbody>
+			{#each AccepedExtensions as item}
+				<tr>
+					<td><a href={item.href}>{item.name}</a></td>
+					<td>{item.extensions}</td>
+				</tr>
+			{/each}
+		</tbody>
+	</table>
+</div>
+
+<style lang="scss">
+	@use '../../../../styles/base-minimal.min.css';
```

Comment:
don't use UNDP design CSS directly in GeoHub repo.

I think here you don't need UNDP deisgn system. Why don't you just use Bulma?